.husky/pre-commit

```diff
@@ -2,8 +2,8 @@
 . "$(dirname -- "$0")/_/husky.sh"
 echo "lint code"
 cd ./src/backend
-npm run precommit
+npm run lint && npx lint-staged
```

Comment:
not sure why but this pre-commit command does not run anymore in my local.

Reply:
It works on my system. I doubt we need to change it for all.

Reply:
yes, i will rollback, it works now